EventListener/PsrResponseListener.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Symfony\Bridge\PsrHttpMessage\EventListener;
+
+use Psr\Http\Message\ResponseInterface;
+use Symfony\Bridge\PsrHttpMessage\HttpFoundationFactoryInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\ViewEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+/**
+ * Converts PSR-7 Response to HttpFoundation Response using the bridge.
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ * @author Alexander M. Turek <me@derrabus.de>
+ */
+final class PsrResponseListener implements EventSubscriberInterface
+{
+    private $httpFoundationFactory;
+
+    public function __construct(HttpFoundationFactoryInterface $httpFoundationFactory)
```

Comment:
same comment as https://github.com/symfony/psr-http-message-bridge/pull/89/files#r576990713
(which I swapped)
can we do without the need for services tat mapto the interface?

Reply:
I think we should apply this change to this class:
```diff
--- a/EventListener/PsrResponseListener.php
+++ b/EventListener/PsrResponseListener.php
@@ -3,6 +3,7 @@
 namespace Symfony\Bridge\PsrHttpMessage\EventListener;
 
 use Psr\Http\Message\ResponseInterface;
+use Symfony\Bridge\PsrHttpMessage\Factory\HttpFoundationFactory;
 use Symfony\Bridge\PsrHttpMessage\HttpFoundationFactoryInterface;
 use Symfony\Component\EventDispatcher\EventSubscriberInterface;
 use Symfony\Component\HttpKernel\Event\ViewEvent;
@@ -18,9 +19,9 @@ final class PsrResponseListener implements EventSubscriberInterface
 {
     private $httpFoundationFactory;
 
-    public function __construct(HttpFoundationFactoryInterface $httpFoundationFactory)
+    public function __construct(HttpFoundationFactoryInterface $httpFoundationFactory = null)
     {
-        $this->httpFoundationFactory = $httpFoundationFactory;
+        $this->httpFoundationFactory = $httpFoundationFactory ?? new HttpFoundationFactory();
     }
 
     /**
```

Reply:
Done. I've also adjusted the constructor call in `PsrResponseListenerTest` accordingly.